hack/upgrade-test-index-image.sh

```diff
@@ -206,6 +224,14 @@ ${CMD} get pod $HCO_CATALOGSOURCE_POD -n ${HCO_CATALOG_NAMESPACE} -o yaml | grep
 
 dump_sccs_after
 
+Msg "make sure that the VM is still running, after the upgrade"
```

Comment:
I think we have also to add a bit of logic to ensure that the VM didn't got restarted in the middle

Reply:
we can do this with a second PR